aptos-move/framework/aptos-framework/sources/vesting.move

```diff
@@ -1151,6 +1152,7 @@ module aptos_framework::vesting {
     }
 
     fun verify_admin(admin: &signer, vesting_contract: &VestingContract) {
+        permissioned_signer::assert_master_signer(admin);
```

Comment:
Could you explain why this assertion is added here and nowhere else?

Reply:
Because all other functions will invoke verify_admin and thus make sure the admin have the right permission configuration.